docs/user-guide/identities/gpg.md

```diff
@@ -0,0 +1,64 @@
+# PGP keys helper
+
+## Why to use PGP?
+
+By default our [Leverage Reference Architectre base-identities layer](https://github.com/binbashar/le-tf-infra-aws/blob/master/security/base-identities/users.tf) 
+approach is to use [IAM module]([https://github.com/binbashar/terraform-aws-iam/tree/master/modules/iam-user]) 
+to manage AWS IAM Users credentials with **encryption to grant strong security**. 
+
+This **module** outputs commands and PGP messages which can be decrypted either using command line to get AWS Web Connsole
+user's password and user's secret key.   
+
+!!! warning "Notes for keybase users"
+    If possible, always use PGP encryption to prevent Terraform from keeping unencrypted password and access secret key 
+    in state file.
+
+!!! check "Keybase pre-requisits"
```

Comment:
Here it should be "pre-requisites" 😉

Reply:
@diego-ojeda-binbash OJ I've just correted this :) Thanks!